test/smoketests/spectests/CMakeLists.txt

```
@@ -72,6 +72,16 @@ set_tests_properties(
     PASS_REGULAR_EXPRESSION "PASSED 4, FAILED 22, SKIPPED 2"
 )
 
+add_test(
+        NAME fizzy/smoketests/spectests/failures-output
+        COMMAND fizzy-spectests ${CMAKE_CURRENT_LIST_DIR}/failures
+)
+set_tests_properties(
+        fizzy/smoketests/spectests/failures-output
+        PROPERTIES
+        PASS_REGULAR_EXPRESSION "FAILED Incorrect returned value\\. Expected: 2 \\(0x2\\) Actual: 10 \\(0xa\\)"
```

```suggestion
        PASS_REGULAR_EXPRESSION "FAILED Incorrect returned value\\. Expected: 12 \\(0xc\\) Actual: 10 \\(0xa\\)"
```

So that both expected and actual are >9.